public/docs/js/latest/guide/displaying-data.jade

```diff
@@ -171,16 +165,16 @@
         that Angular knows to include it:
```

Comment:
L164: should that just say "the angular <code>For</code> directive" to be consistent with the new TS example?